docs/index.asciidoc

```diff
@@ -222,5 +223,15 @@ of your ruby script file defined in `path`.
 Tag to add to events in case the ruby code (either inline or file based)
 causes an exception.
 
+[id="plugins-{type}s-{plugin}-tag_with_exception_message"]
+===== `tag_with_exception_message`
+
+* Value type is <<boolean,boolean>>
+* Default value is `false`
+
+If `true` adds a tag to the event that's the concatenation of <<plugins-{type}s-{plugin}-tag_with_exception_message>>
```

Comment:
```suggestion
If `true` adds a tag to the event that is the concatenation of `tag_with_exception_message`
```

Reply:
Let's hard code this. A link inside the section it's linking to doesn't make sense. 

<img width="840" alt="Screen Shot 2021-06-01 at 7 10 59 PM" src="https://user-images.githubusercontent.com/35154725/120401378-d2967900-c30d-11eb-86c8-e2b3e1a95951.png">